filebeat/docs/modules/traefik.asciidoc

```diff
@@ -3,9 +3,66 @@ This file is generated! See scripts/docs_collector.py
 ////
 
 [[filebeat-module-traefik]]
+:modulename: traefik
+
 == Traefik module
 
-This is the traefik module.
+The +{modulename}+ module parses access logs created by
+https://traefik.io/[Træfik].
+
+include::../include/what-happens.asciidoc[]
+
+[float]
+=== Compatibility
+
+This module requires the
+{elasticsearch-plugins}/ingest-user-agent.html[ingest-user-agent] and
+{elasticsearch-plugins}/ingest-geoip.html[ingest-geoip] Elasticsearch plugins.
+
+//REVIEWERS: Do we need to say anything else about compatibility here?
+
+include::../include/running-modules.asciidoc[]
+
+[float]
+=== Example dashboards
+
+This module comes with sample dashboards. For example:
+
+[role="screenshot"]
+image::./images/kibana-traefik.png[]
```

Comment:
Jenkins is saying: 

> ERROR: missing resource: /var/lib/jenkins/workspace/elastic+beats+pull-request+multijob-docs/src/github.com/elastic/beats/filebeat/docs/images/kibana-traefik.png

Reply:
@andrewkroh whoops forgot to add the image. I am getting git sloppy these days. :-/